README.md

```diff
@@ -1 +1,10 @@
-# twitter-clone-mariam
\ No newline at end of file
+
+# twitter-clone-mariam
+ oreo website  :A social networking site that allows a user to have short posts of up to 300 characters called tweets. It also allows the user to send messages between his friends. The user can follow other people and see their tweets, like them and share them.
```

Comment:
You should describe your website, which is a clone of Twitter 
I mean describe yours and what features you will build not the whole Twitter app!

Reply:
so, Does "oreo" will allow users to send messages between them? or send follow request!?